src/any_agent/frameworks/any_agent.py

```diff
@@ -47,11 +52,21 @@ def create(
             msg = f"Unsupported agent framework: {agent_framework}"
             raise ValueError(msg)
         agent = Agent(agent_config, managed_agents=managed_agents)
-        asyncio.get_event_loop().run_until_complete(agent._load_agent())
+
+        if tracing:
+            # Agno not yet supported https://github.com/Arize-ai/openinference/issues/1302
+            # Google ADK not yet supported https://github.com/Arize-ai/openinference/issues/1506
+            if agent_framework in (AgentFramework.AGNO, AgentFramework.GOOGLE):
+                logger.warning(
+                    "Tracing is not yet supported for AGNO and GOOGLE frameworks. "
+                )
+            else:
+                agent.trace_filepath = setup_tracing(agent_framework, tracing)
+        asyncio.get_event_loop().run_until_complete(agent.load_agent())
         return agent
 
     @abstractmethod
-    async def _load_agent(self) -> None:
+    async def load_agent(self) -> None:
```

Comment:
I guess this is unrelated (but I agree is a good change).

We should document the API for people using `any_agent` within async code

Reply:
https://github.com/mozilla-ai/any-agent/issues/121